app/models/patient.rb

```diff
@@ -708,9 +712,20 @@ def latest_transfer
   #     AND
   #  - on the last day of or past their monitoring period
   scope :close_eligible, lambda {
-    exposure_asymptomatic
-      .submitted_assessment_today
-      .end_of_monitoring_period
+    where(isolation: false)
+      .non_reporting
+      .no_recent_activity
+      .or(
+        exposure_asymptomatic
+        .submitted_assessment_today
+        .end_of_monitoring_period
+      )
+  }
+
+  # If a patient record has been inactive for 30 days or more,
+  # then it should be automatically closed as part of the close patients job.
+  scope :no_recent_activity, lambda {
+    where('updated_at <= ?', 30.days.ago)
```

Comment:
For this scope that is used to see if the monitoree is eligible for auto-closing, I see it checks the monitoree's `updated_at` field. Do you know if this field is touched if the Sara Alert system sends the monitoree a symptom assessment notification that results in a history entry being created on the monitoree? 

Thankfully this won't be an issue for the monitorees on the Non-Reporting Exposure line list since they will stop receiving symptom assessments after the normal 14-day monitoring period. But if this scope were to be used for monitorees in a different line list where that is not the case, we'll have to be careful of this possibility.

Reply:
Sending an assessment will change the updated_at field. The intention for this specific ticket was just for non-reporting in the exposure workflow, but if that intention ends up changing then this logic would need to change like you mentioned.